### PR TITLE
Update muzzle to 1.4

### PR DESCRIPTION
**Pull request by suschizu**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] The commit message includes the cask’s name and version.

### CODE REVIEW

File: Casks/muzzle.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 cask 'muzzle' do
-  version '1.3'
-  sha256 'e76e5f5358e8749e7fe2294fee3717ac7327e79eff227430bf3d8f601684a0e6'
+  version '1.4'
+  sha256 '5c3b608f836878aba78a01ee69d2efd4951b644b5f21075648d10ad04223b9fb'
```
Comment (mybrainishuge) [inline Casks/muzzle.rb]: @victorpopkov Is this sha right? I'm getting a complaint from brew:
```
Expected: 5c3b608f836878aba78a01ee69d2efd4951b644b5f21075648d10ad04223b9fb
Actual:   2f3c3c839787408d5b991651b5ba25323a12904b4d155e8074f68e7afea3f845
```

Comment (suschizu) [reply Casks/muzzle.rb]: @mybrainishuge - there was an update yesterday to version 1.5 that has the actual checksum - just made a pull request

Comment (suschizu) [reply Casks/muzzle.rb]: 1.4 seems to be a mistake here: https://muzzleapp.com/updates/sparkle-version-history.html  
as you can see in the appcast the newest download-url refers to buildnr. 304 and shortVersionString 1.5

Comment (mybrainishuge) [reply Casks/muzzle.rb]: That makes sense. I checked the site which says 1.4 for the last two versions. I’m guessing the latest entry is a typo in the release notes. Thanks!

**Pull request merged**